04LessonFour/LessonFour.js

```diff
@@ -2,38 +2,47 @@ module.exports = function() {
 
   this.arrayPop = function (arr) {
     // should remove the last item in the array, then return the arr
-
+    arr.pop();
+    return arr;
   }
 
   this.arrayPush = function (arr, item1, item2, item3) {
     // should add these items onto the end of the array, then return the array
-
+    arr.push(item1, item2, item3);
+    return arr;
   }
 
   this.arrayShift = function (arr) {
     // should remove the first item in the array, then return the array
-
+    arr.shift();
+    return arr;
   }
 
   this.arrayUnshift = function (arr, item1, item2, item3) {
     // should add these items onto the front of the array, then return the array
-
+    arr.unshift(item1, item2, item3);
+    return arr;
   }
 
   this.createObject = function() {
     // should return an object with keys 'first', 'second', 'third' mapped to
     // values 1, 2, 3
-
+    var object1 = new Object();
+    object1['first'] = 1;
+    object1['second'] = 2;
+    object1['third'] = 3;
+    return object1;
   }
```

Comment:
This works. Are you also able to create an object with all these keys and values using a single assignment instead of 4?